### ISSUE TITLE
Hi!I have a better suggest  for AuthMe plugin.

### ISSUE THREAD

**Issue by You0Hun** [Xephi/AuthMeReloaded#865]
I have a better suggest  for AuthMe plugin.
Please tell me your mail.
So that I can send you pictures.
The picture is about the plugin.
My mail: you0hun@gmail.com

**Comment by ljacqu** [comment]
Hi, you can upload the picture e.g. on http://postimage.org

**Comment by sgdc3** [comment]
@You0Hun

**Comment by You0Hun** [comment]
https://s31.postimg.org/uut4cl4qj/W04_45_P_2_1_W3_FBS_Q8.png

**Comment by You0Hun** [comment]
This picture is my preliminary design！Hope AuthMe can have this function！[url=https://postimg.org/image/z3xuer7zr/][img]https://s31.postimg.org/z3xuer7zr/W04_45_P_2_1_W3_FBS_Q8.png[/img][/url]
https://s31.postimg.org/uut4cl4qj/W04_45_P_2_1_W3_FBS_Q8.png
https://postimg.org/image/z3xuer7zr/

**Comment by You0Hun** [comment]
Can you see the picture?

**Comment by You0Hun** [comment]
If you can't add this feature！So please tell me how to call the AuthMe inside the login！I will thank you very much！

**Comment by You0Hun** [comment]
By the way,I come from China.English is not very good.Cannot read please forgive me.

**Comment by You0Hun** [comment]
https://s26.postimg.org/cmnwhb9qx/W04_45_P_2_1_W3_FBS_Q8.png

**Comment by sgdc3** [comment]
Hello, probably we should provide a "login strategy" api @ljacqu

**Comment by You0Hun** [comment]
"login strategy" ??

**Comment by sgdc3** [comment]
like:

player join -> authme handle join -> [Custom login strategy] -> authme handle login

**Comment by ljacqu** [comment]
@sgdc3 I just skimmed through this, but wouldn't some plugin on top of AuthMe already be able to do this? By setting the login message to empty AuthMe won't output any "Please /login" message and the plugin on top can then fire "/login pass" based on the user's input, or use the API

**Comment by sgdc3** [comment]
@ljacqu yeah, but it's not the first time someone want to modify the login procedure

**Comment by sgdc3** [comment]
Someone for example asked to use a holo interface

**Comment by sgdc3** [comment]
or a virtual sign as input field

**Comment by You0Hun** [comment]
..

**Comment by RikoDEV** [comment]
Good option!
